### DISCUSSION TITLE
My wishes for media library improvement

### DISCUSSION THREAD

**Discussion by DarkTraveler** [strapi/strapi#5828] [Legacy Topics]
Hello. The beta.20's update is great!

I have already tested your new media library.
And I want to share my opinion:

1) The Media Library does not have the ability to create folders for grouping files. All pictures in one heap. It is not comfortable.
2) The media library popup is small, although I have a full-HD screen. I would like to add the ability to stretch it, or at least make it taller, because very few images are placed in the visible area at a time.

**Comment by DarkTraveler** [comment]
Is there anyone alive?)

**Comment by lauriejim** [comment]
Hello hello! 
Yes, of course, we are alive, just loooot of things to manage and it doesn't look like but we still a small team to manage all this activity on GitHub.
I shared this with the team. About the **folder** topic, we have a related issue here - https://github.com/strapi/strapi/issues/2972

**Comment by Aurelsicoko** [comment]
Hi @DarkWayfarer, thank you so much for your feedback! I'm glad to see that you enjoy the new media library. I'll try my best to answer your questions:

- **Folders**: we decided to not use folders structure for the v1. The main goals were to be able to reuse media in multiple entries, be SEO and responsive-friendly, and offer basic image manipulation. I would also like to see our media library supporting folders, we'll do it in another iteration.

- **Popup**: Having a popup has a lot of advantages, you keep the same context in your screen, you don't go to another page, etc. However, it also adds complexity to make sure the popup is using the screen size at its best. The popup is currently a bit small (even if it's larger than it used to be) and I would love to see the popup also responsive. Currently, the size (width/height) is hardcoded. As the first answer, it's something we can easily improve in a next iteration.

Thus, I completely agree with you and your feedback is really great! We have to make choices and we can't spend months on each feature. Right now, the top priority is the release of the stable and the development of the Role-Based Access Control.
